scripts/runalltests.sh

```diff
@@ -27,13 +27,20 @@ cd test
 echo "Running all tests:"
 echo "cpp-ethereum HEAD "$cppHead
 echo "tests HEAD "$testHead
+exec 2> $workdir/testerror.txt
+timestart=$(date +%s.%N)
 ./testeth --all --exectimelog
+timeend=$(date +%s.%N)
 
-exec &> /dev/null
-#cd $workdir
-#rm results.zip
-#zip results.zip *.txt
-date=$(date +%Y-%m-%d)
+#Prepare Header of the report
+exec &> $workdir/testreport.txt
+exectime=$(echo "$timeend - $timestart" | bc)
+echo "REPORT"
+echo "Test execution time: "$exectime
+cat $workdir/testerror.txt
+cat $workdir/testreport.txt | cat - $workdir/testlog.txt > temp && mv temp $workdir/testlog.txt
```

Comment:
Wouldn't `cat $workdir/testreport.txt $workdir/testlog.txt > $workdir/testlog.txt` do the same?